PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -1276,34 +1276,24 @@ - (void)toggleAnnotationToolbar:(CDVInvokedUrlCommand *)command
 
 #pragma mark Delegate methods
 
-- (BOOL)pdfViewController:(PSPDFViewController *)pdfController shouldScrollToPage:(NSUInteger)page
+- (void)pdfViewController:(PSPDFViewController *)pdfController willBeginDisplayingPageView:(PSPDFPageView *)pageView forPageAtIndex:(NSInteger)pageIndex
 {
-    return [self sendEventWithJSON:[NSString stringWithFormat:@"{type:'shouldScrollToPage',page:%ld}", (long)page]];
+    [self sendEventWithJSON:[NSString stringWithFormat:@"{type:'willBeginDisplayingPageView',page:%ld}", (long) pageView.pageIndex]];
```

Comment:
You can use `pageIndex` instead of `pageView.pageIndex` to pass to the JSON event.